hometask3/src/pagesFactory.ts

```diff
@@ -0,0 +1,15 @@
+import Page from "./page";
+
+export const createPages = (numberOfPages: number, pageType: string) => {
+  let settings: [string, string] = ['with text','simple paper'];
+  switch (pageType) {
```

Comment:
Personally I prefer not to use `switch`. I'd better have something like this:

```
const settings = {
    magazine: ["with article", "glossy paper"],
    comics: ["with images", "glossy paper"]
}

const pageSettings = settings[pageType] || ['with text','simple paper']
```

Furthermore, in this case as long as `settings` is static object, we should move it to global scope from function scope.